src/ex1/script.js

```diff
@@ -0,0 +1,91 @@
+let addTaskBtn = document.querySelector(".submitBtn")
+let inputText = document.querySelector(".taskText")
+
+//Handle Image
+function addImg() {
```

Comment:
addImg is not a clear name for a function - addEmptyStateImage would be more readable